business_requirement_deliverable_cost/models/business.py

```diff
@@ -157,6 +157,14 @@ def action_button_update_estimation(self):
                     resource._set_sales_price()
                     resource._set_cost_price()
 
+    @api.multi
+    def action_button_update_total_revenue(self):
+        total_revenue = 0.0
+        if self.price_total != total_revenue:
+            for rl in self.resource_ids:
```

Comment:
@sudhir-serpentcs Think you could use https://docs.python.org/2/library/functions.html#map function.